.github/workflows/wheels.yml

```diff
@@ -105,10 +110,17 @@ jobs:
             py: cp310
             arch: universal2
 
+          # Windows-latest cp310, we can only include win_amd64 builds as numpy
+          # does not support win
+          # Please see https://cibuildwheel.readthedocs.io/en/stable/options/#archs
+          - os: windows-latest
+            py: cp310
```

Comment:
Can we support older python versions on windows?

Reply:
Yup, that's already included above with `os: windows-latest, arch: [auto]`.

Essentially I only remove `cp10-win32` but keep all the others, take a look at the link
https://cibuildwheel.readthedocs.io/en/stable/options/#archs and the option there for `CIBW_BUILD`.